allauth/socialaccount/providers/openid_connect/provider.py

```diff
@@ -64,9 +64,10 @@ def extract_common_fields(self, data):
         return dict(
             email=data.get("email"),
             username=data.get("preferred_username"),
-            name=data.get("name"),
```

Comment:
`name` is actually used in `adapter.populate_user()` as a fallback in case `first_name` / `last_name` are empty. Given that some OpenID Connect providers may only use `name`, and some only `first_name` / `last_name`, I think it's best to keep this line?